component/monitoring.libsonnet

```diff
@@ -0,0 +1,120 @@
+/*
+This file adds ServiceMonitors in combination with the Prometheus component.
+
+Warning: The operators monitored are instance independent,
+         everything in this file must be instance independent.
+*/
+
+local cm = import 'lib/cert-manager.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local prom = import 'lib/prometheus.libsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.openshift4_operators;
+
+local nsName = 'syn-monitoring-openshift4-operators';
+
+local promInstance =
+  if params.monitoring.instance != null then
+    params.monitoring.instance
+  else
+    inv.parameters.prometheus.defaultInstance;
+
+local endpoint = function(serverName)
+  prom.ServiceMonitorHttpsEndpoint(serverName) {
+    port: 'https-metrics',
+    metricRelabelings: [
+      {
+        action: 'drop',
+        regex: 'etcd_(debugging|disk|request|server).*',
+        sourceLabels: [
+          '__name__',
+        ],
+      },
+      prom.DropRuntimeMetrics,
+    ],
+  };
+
+local marketplaceOperatorMonitor = prom.ServiceMonitor('openshift-marketplace-operator') {
```

Comment:
According to the internal ticket the marketplace operator only provides Go runtime metrics, do we really need a ServiceMonitor for it which drops all metrics?